examples/lib/stories/games/padracing/game_colors.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flame/extensions.dart';
+import 'package:flutter/material.dart' hide Image, Gradient;
+
+enum GameColors {
+  green,
+  blue,
+}
+
+extension GameColorExtension on GameColors {
```

Comment:
We can use enum properties now :D

Reply:
We can't, we are not on 3.0.0 yet (after v1.2 is released we'll upgrade)

Reply:
Oh right! For a moment I thought we already had migrated